src/api/app/controllers/source_controller.rb

```diff
@@ -568,9 +568,17 @@ def check_and_remove_repositories!(repositories, full_remove, force = false)
 
   # GET /source/:project/_config
   def show_project_config
-    path = request.path_info
-    path += build_query_from_hash(params, [:rev])
-    pass_to_backend path
+    forward_from_backend(BackendFile.query_from_list(params, [:rev]))
+
+    project = Project.get_by_name(params[:project])
+    content = project.config.to_s
```

Comment:
I think that Project.get_by_name tries to find remote project also... it calls find_remote_project at line 308:
```
      dbp, remote_name = find_remote_project(name)
```

But, will crash if Project doesn't exists or user hadn't access to that. I have to surround that with a rescue and show the exception in a proper way.